redbot/core/bot.py

```diff
@@ -10,7 +10,7 @@
 from enum import IntEnum
 from importlib.machinery import ModuleSpec
 from pathlib import Path
-from typing import Optional, Union, List, Dict, NoReturn
+from typing import Optional, Union, List, Dict, NoReturn, Set, Coroutine, TypeVar
```

Comment:
```suggestion
from typing import (
    Any,
    Awaitable,
    Callable,
    Coroutine,
    Dict,
    List,
    NoReturn,
    Optional,
    Set,
    TypeVar,
    Union,
)
```
Changed imports assuming the suggestion below makes sense.